frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte

```diff
@@ -32,9 +33,9 @@ to reveal and hide the password
 -->
 
 <div class="relative">
-  <label for="password" class="hidden">{$t('candidate.password')}</label>
+  <label for={id} class="hidden">{$t('candidate.password')}</label>
   <input
-    id="password"
+    id={password}
```

Comment:
The id of an html element should be constant. Therefore, the id can't be the password value. Also, this is a security hazard, as the id, which would be the user's password, is visible to the dom, meaning that it would be trivial for a potential attacker to obtain it. 

It think it should just say:
`{id}`